projects/plugins/jetpack/extensions/blocks/nextdoor/index.js

```diff
@@ -0,0 +1,59 @@
+import { SocialServiceIcon } from '@automattic/jetpack-components';
+import { createBlock } from '@wordpress/blocks';
+import { __, _x } from '@wordpress/i18n';
+import { getIconColor } from '../../shared/block-icons';
+import attributes from './attributes';
+import edit from './edit';
+import save from './save';
+import './editor.scss';
+import { REGEX, parseUrl } from './utils';
+
+export const name = 'nextdoor';
```

Comment:
Given that we're moving towards `block.json` to declare blocks, do you think you could take a similar approach for this new block?

You can take a look at 8447ef5075727869218037724844a6b6ba29b35c to see how the change was done for the Calendly block, for example.